src/modules/wlr/taskbar.cpp

```diff
@@ -182,11 +183,21 @@ bool Task::image_load_icon(Gtk::Image &image, const Glib::RefPtr<Gtk::IconTheme>
 
   try {
     pixbuf = icon_theme->load_icon(ret_icon_name, scaled_icon_size, Gtk::ICON_LOOKUP_FORCE_SIZE);
+    spdlog::debug("{} Loaded icon '{}'", repr(), ret_icon_name);
   } catch (...) {
-    if (Glib::file_test(ret_icon_name, Glib::FILE_TEST_EXISTS))
+    if (Glib::file_test(ret_icon_name, Glib::FILE_TEST_EXISTS)) {
       pixbuf = load_icon_from_file(ret_icon_name, scaled_icon_size);
-    else
-      pixbuf = {};
+      spdlog::debug("{} Loaded icon from file '{}'", repr(), ret_icon_name);
+    } else {
+      try {
+        pixbuf = DefaultGtkIconThemeWrapper::load_icon("image-missing", scaled_icon_size,
+            Gtk::IconLookupFlags::ICON_LOOKUP_FORCE_SIZE);
```

Comment:
The "image-missing" icon is maybe not the prettiest, something like [this](https://commons.wikimedia.org/wiki/File:Application-default-icon.svg) maybe looks prettier. A different approach is thus to bundle in a fallback icon in resources/icons (and update resources/icons/waybar_icons.gresource.xml).

This line is then replaced by `pixbuf = Gdk::Pixbuf::create_from_resource("/fr/arouillard/waybar/icons/scalable/apps/default-app-icon", scaled_icon_size, scaled_icon_size)`

If bundling an icon preferred, I am happy to use this approach instead and change the PR accordingly.